samples/tab-approval/csharp/README.md

```diff
@@ -0,0 +1,139 @@
+﻿# Tab request approval with activity feed notification.
+
+This sample shows a feature where:
+1)Requester: Can request for any task to the manager by sending activity feed notification and can see his request status.
```

Comment:
Can request for any task approval to the manager by sending activity feed notification and can see his request status.